packages/react/src/components/Table/Table.jsx

```diff
@@ -991,6 +991,7 @@ const Table = (props) => {
                   'hasRowNesting',
                   'hasSingleRowEdit',
                   'hasRowSelection',
+                  'showRadioButton',
```

Comment:
we may want to change the name of this prop to be more informative on where we are showing the radio button. `useRadioButtonSingleSelect` or something like that.